basics.js

```diff
@@ -42,15 +58,16 @@ var anna = {name: "Anna",
           }
 
 // What is `anna`? (e.g. method, function, string, etc.) 
-
+// variable
 
 // What are `name`, `age`, and `luckyNumber`? 
-
+// properties of the object/variable "anna"
 
 // What does `this` refer to? 
-
+// 'this' refers to the object/variable that it's scoped to, so in this case it's 'anna'
 
 // If you want to return the value of anna's lucky number, how would you do that? 
+// this.luckyNumber or anna.luckyNumber
```

Comment:
Make sure you put () at the end of the method name, otherwise it will just return the definition of the method.